.travis.yml

```diff
@@ -73,3 +73,19 @@ jobs:
         - OPENWHISK_HOST="172.17.0.1" USERS="1" REQUESTS_PER_SEC="1" ./gradlew gatlingRun-ColdBlockingInvokeSimulation
       env:
         - DESCRIPTION="Execute wrk-performance test suite."
+    - script:
```

Comment:
Adding 2 more jobs would lead to more resource usage and thus may exhaust the travis pool. May be we look into running these suites as a cron and not for every PR.

Reply:
Thanks, can you point me to the place where I can configure it as a cron job?

Reply:
This can be done by Travis Settings and we can then detect the [cron event type][1] in travis config to [execute job conditionally][2]

[1]: https://docs.travis-ci.com/user/cron-jobs/#detecting-builds-triggered-by-cron
[2]: https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-jobs

Reply:
I agree with @chetanmeh; lets make running the lean test configurations conditional/disabled at least initially.